lib/orderbook/OrderBook.ts

```diff
@@ -25,8 +25,14 @@ class OrderBook extends EventEmitter {
   private repository: OrderBookRepository;
   private matchesProcessor: MatchesProcessor = new MatchesProcessor();
 
-  private ownOrders: { [ pairId: string ]: Orders } = {};
-  private peerOrders: { [ pairId: string ]: Orders } = {};
+  private ownOrders: Map<String, Orders> = new Map<String, Orders>();
+  private peerOrders: Map<String, Orders> = new Map<String, Orders>();
+
+  /**
+   * This map has the localId of and order as key and the global id as value
+   * which makes possible to cancel orders by their local id without iterating
+   */
```

Comment:
Maybe let's write a more general description, something like: A map between an order localId and id (global).